### PR TITLE
modify batch_norm and batch_norm_grad. *test=kunlun

### PR DESCRIPTION
**Pull request by Zhangjingyu06**
### PR types
Bug fixes

### PR changes
Ops

### Describe
modify batch norm and batch norm grad

### CODE REVIEW

**Review by ykkk2333 — APPROVED** [review]
LGTM

**Review by ZibinGuo — APPROVED** [review]
LGTM

**Review by QingshuChen — APPROVED** [review]
LGTM
